### PR TITLE
Multi OS support on CloudBreak

### PR DESCRIPTION
**Pull request by leki75**
Multi OS support on CloudBreak

* hardcoded eth0 interface changed to default ethernet interface
* salt-bootstrap version upgrade: ubuntu16 fixes

### CODE REVIEW

**Comment by tibkiss** [comment]
+1

File: saltstack/base/salt/salt-bootstrap/init.sls [hunk 1]
```diff
@@ -1,7 +1,7 @@
 install_saltbootstrap:
   archive.extracted:
     - name: /usr/sbin/
-    - source: https://github.com/hortonworks/salt-bootstrap/releases/download/v0.12.0/salt-bootstrap_0.12.0_Linux_x86_64.tgz
+    - source: https://github.com/hortonworks/salt-bootstrap/releases/download/v0.12.0/salt-bootstrap_0.12.1_Linux_x86_64.tgz
```
Comment (lacikaaa) [inline saltstack/base/salt/salt-bootstrap/init.sls]: this link is not valid: `https://github.com/hortonworks/salt-bootstrap/releases/download/v0.12.0/salt-bootstrap_0.12.1_Linux_x86_64.tgz`
replace with v0.12.1 the part after download

Comment (leki75) [reply saltstack/base/salt/salt-bootstrap/init.sls]: Thanks for checking it.

Comment (leki75) [reply saltstack/base/salt/salt-bootstrap/init.sls]: Fixed

**Pull request merged**